### DISCUSSION TITLE
Default shared entries in the meal plan

### DISCUSSION THREAD

**Discussion by sortab** [TandoorRecipes/recipes#1651] [Q&A] [answered]
When making an entry in the meal plan I have to share it with my wife's user. Anyway to change the default behavior so my entries in the meal plan are shared with her, and vice versa?

**Comment by smilerz** [comment, marked as answer]
User user preferences you can save a default shared user.  (and default shared shopping list)

**Reply by sortab** [reply]
OK, took a little digging but I found it (under my own user preferences I can select another user with whom I by default share my meal plans). Thanks (again) for the help.
